### DISCUSSION TITLE
Heyo! got a quick question

### DISCUSSION THREAD

**Discussion by ISmileEveryday** [HaloOfBlocks/AdditionalGuns#83] [Questions & Answers] [answered]
are there any plans to improve the workbench menu perhaps? its kind of a nightmare with the current setup to find the gun youre looking for. keep up the great work tho, love the mod!

**Comment by Autovw** [comment, marked as answer]
Glad to hear you like the mod!

We do have plans for a custom Workbench (and there will be one before we even consider the mod to be "stable") but I can't give you an ETA on that.

**Reply by ISmileEveryday** [reply]
thanks for the quick answer man! 
can we expect more updates or is the mod in development until "stable"?

**Reply by Autovw** [reply]
Yes, of course! 0.8.0 is currently in the works :)

**Reply by CasualNoLifer** [reply]
> 

 Just to add on, you can view some entries on the Discussions Page to see some upcoming plans for additions. :)
